lib/trebuchet/feature.rb

```diff
@@ -133,6 +134,12 @@ def set_expiration_date(expiration_date)
     Trebuchet.backend.set_expiration_date(self.name, expiration_date)
   end
 
+  # Set the feature to be force depoyed.
```

Comment:
i don't think this comment adds value (this is what i'd expect from a function called "set force") as is ... but what does it actually mean to be force deployed? would you be able to capture that in a 1-2 line comment?